src/app/modules/library/select-all-projects-checkbox/select-all-projects-checkbox.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-checkbox
+  color="primary"
+  [checked]="selectedAllProjects"
+  [indeterminate]="selectedSomeProjects"
+  (click)="selectAllProjectsCheckboxClicked()"
```

Comment:
Do we need the function name to contain all projects and checkbox and clicked, or is "select()" clear enough because of the context?
```suggestion
  (click)="select()"
```